ooiui/static/js/views/common/SVGView.js

```diff
@@ -0,0 +1,48 @@
+var SVGView = Backbone.View.extend({
+  initialize: function(options) {
+    if(options && options.url) {
+      this.url = options.url;
+      this.fetch();
+    } else {
+      console.error("SVG View needs a url");
+    }
+  },
+  fetch: function() {
+    var self = this;
+    $.get(this.url,
+      function(svgDoc) {
+        var importedSVGRootElement = document.importNode(svgDoc.documentElement,true);
+        self.$el.html(importedSVGRootElement);
+        self.render();
+      },
+      "xml");
+  },
+  render: function() {
+  }
+});
+
+var SVGPlotView = SVGView.extend({
+  render: function() {
+    var self = this;
+    _.each(this.$el.find('#axes_1').children(), function(child) {
+      if(child.id.match(/PathCollection_[0-9]+$/)) {
+        if(child.id != 'PathCollection_1') {
+          self.$el.find(child).remove();
+        }
+      }
+    });
```

Comment:
Unfortunately Matplotlib Has 10 groups of the exact same points for basic scatter plots. Here, I use jquery to remove the extras, and leave only `#PathCollection_1`